### PR TITLE
docs: Update reserved-ip.md

### PR DESCRIPTION
**Pull request by iiiceoo**
Signed-off-by: iiiceoo <ziqian.xue@daocloud.io>

### CODE REVIEW

File: docs/usage/reserved-ip.md [hunk 1]
```diff
@@ -130,25 +127,31 @@ reservedip-deploy-6cf9858886-lb7cr   0/1     ContainerCreating   0          35s
...
+-f https://raw.githubusercontent.com/spidernet-io/spiderpool/main/docs/example/reserved-ip/test-ipv4-reservedip.yaml \
+-f https://raw.githubusercontent.com/spidernet-io/spiderpool/main/docs/example/reserved-ip/test-ipv4-ippool.yaml \
+-f https://raw.githubusercontent.com/spidernet-io/spiderpool/main/docs/example/reserved-ip/reservedip-deploy.yaml \
+--ignore-not-found=true
 ```
 
 ## A Trap
 
-So, can you use SpiderIPPool's field `excludeIPs` to achieve the same effect as SpiderReservedIP? The answer is **NO**! Look at such a case, now you want to reserve an IP `172.18.43.31` for an external application of the cluster, which may be a Redis node. To achieve this, you created such an SpiderIPPool:
+So, can we use IPPool's field `excludeIPs` to achieve the same effect as ReservedIP? The answer is **NO**! Look at such a case, now we want to reserve an IP `172.18.43.31` for an external application of the cluster, which may be a Redis node. To achieve this, we should create such an IPPool:
```
Comment (windsonsea) [inline docs/usage/reserved-ip.md:154]: `we` is not suggested in a user manual.

**Review by windsonsea — COMMENTED** [review]
As a user manual, the purpose is to tell how your users can get the results step by step.

Avoid to use `we` is a skill to tell, refer to [some writing skills](https://online-learning.com/five-tips-writing-user-manual/#:~:text=User%20manuals%20should%20always%20focus,you%E2%80%9D%20and%20the%20active%20voice).

For example, you can find some difference below:

1. 我们搬开了桌子，拉出了抽屉，找到了钥匙。
2. 你搬开桌子，拉出抽屉，就能找到钥匙。

> 1 是陈述一个故事，2 是一个教程步骤。
> 1 是别人的行为，2 是一切为你

File: docs/usage/reserved-ip.md (hunk 1 above)
Comment (iiiceoo) [reply docs/usage/reserved-ip.md:154]: Fine, I will modify it, but I don't want to replace "IPPool" with "SpiderIPPool".

Comment (windsonsea) [reply docs/usage/reserved-ip.md]: 😅  This clause is fine to me. We can modify others later.

**Comment by windsonsea** [comment]
/lgtm

For style issue about `we` and `you`, we need a check through all docs later.

**Pull request merged**